### ISSUE TITLE
spd_utility skips updating "other" dependency types

### ISSUE THREAD

**Issue by btgoodwin** [Geontech/meta-redhawk-sdr#47]
It looks like upstream, REDHAWK uses `other` instead of `runtime_requirements` (at least that's what the IDE is enforcing).  If dependencies aren't patched to also identify the `cpp-PACKAGE_ARCH` location, the autoconf macros will not be able to crawl from one SPD to the next to find the package config files, resulting in a package config error.

This is a few-character fix on line 114 of spd_utility:

```
        if dep.get_type() in ['other', 'runtime_requirements']:
```

**Comment by btgoodwin** [comment]
After conferring with @jirwin-geon, the attribute may not actually be important so we'll simply patch any references we find.

**Comment by btgoodwin** [comment]
Closed by #51 and #52 for thud and rocko, respectively.
